pkg/users/account-create-dialog.js

```diff
@@ -328,15 +400,21 @@
                             "-e",
                             user_name
                         ], { superuser: "require", err: "message" });
+                })
+                .then(() => {
+                    if (force_home) {
+                        return cockpit.spawn(["id", user_name, "--group"], { superuser: "require", err: "message" })
+                                .then(gid => cockpit.spawn(["chown", "-hR", `${user_name}:${gid.trim()}`, home_dir], { superuser: "require", err: "message" }));
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230324-000238-1c01f86d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)